src/common/collection_column_specs/genome_attribs-PMI.yml

```diff
@@ -23,6 +23,21 @@ columns:
      type: string
      filter_strategy: ngram
 
+   - key: kbase_genome_size
```

Comment:
If/when we do display names & categories we'll need updates for these

Reply:
hmm, unclear on what you mean.

Reply:
In the mockup discussions we've talked about adding display name and column categories for the fields. We have a list of those for the current fields, but we'll need them for these fields as well